cmake-modules/AzureCodeCoverage.cmake

```diff
@@ -55,6 +55,8 @@ function(add_gcovr_html)
             -r ${BASEDIR} ${EXCLUDE_ARGS}
             --object-directory=${PROJECT_BINARY_DIR}
             -o ${args_TARGET_NAME}/index.html
+            --exclude-unreachable-branches
+            --exclude-throw-branches
```

Comment:
Are we sure we want to exclude throw branches? We have `EXPECT_THROW` for such testing, no?

Reply:
It is supposed to be for the cases when you pass a string literal to a function that expects `std::string`: theoretically, that might be a moment when your system runs out a last piece of RAM, and `std::string` fails to create, so it might throw.